### DISCUSSION TITLE
🔗 Container Text Styles for Widget Inheritance (Duplicate of #20529)

### DISCUSSION THREAD

**Discussion by danieliser** [elementor/elementor#24780] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I have to add the same text color or styling to many different widgets within the same container. This is not ideal and not well optimized to fully take advantage of C(ascading)SS.

### Describe the solution you'd like.

I could add text color to a container and have all widgets text inherit that value unless overriden at the widget level.

This would drastically reduce duplication of code & settings changes needed when creating a page, and result in much leaner CSS style sheets.

### Describe alternatives you've considered.

Custom CSS, not ideal.

### Additional context

Having switched from another page builder that had this from the beginning, I'm actually quite frustrated this is still not optimized in Elementor this far along.

### Agreement

- [x] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by danieliser** [comment]
Further use case would be a global widget/container inserted into a section or container. You could overload all elements within the global without making changes to it, with text controls on the parent.

**Comment by pumadog** [comment]
Already requested 14 months ago. Please comment there: https://github.com/orgs/elementor/discussions/20529

**Comment by danieliser** [comment]
Added my 2 cents there @pumadog, thanks.

**Comment by nicholaszein** [comment]
Hello there!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #20529

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
